future-apps/payroll/app/src/components/Table/Table.js

```diff
@@ -8,10 +9,29 @@ import TableRow from './TableRow'
 import Panel from '../Panel/Panel'
 import { sort, SORT_DIRECTION } from '../../utils/sorting'
 
+const footerStyles = {
```

Comment:
I think we should use styled-component for this

Reply:
The footer styles were taken out to a single StyledButton styled component